src/client/componentsMealsharing/Meals.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "./Meal.css";
+import { FancyBorder } from "./FancyBorder";
+
+function Meals() {
+  const [fetchData, setFetchData] = useState([]);
```

Comment:
This could have a more descriptive name for example 
```suggestion
  const [meals, setMeals] = useState([]);
```